docs/docusaurus.config.js

```diff
@@ -332,9 +332,9 @@ module.exports = {
                 categoryLinkSource: "auto",
             },
           },
-          webkey_v3: {
-            specPath: ".artifacts/openapi/zitadel/resources/webkey/v3alpha/webkey_service.swagger.json",
-            outputDir: "docs/apis/resources/webkey_service_v3",
+          webkey_v2: {
```

Comment:
Do we leave this under API v3?

Reply:
this is just the docs generation part... the side nav is [here](https://github.com/zitadel/zitadel/pull/9445/files/74b2c368d3ece2960a8b126e9866387d71f72f37#diff-e8aa23ec31a264f87ba92ce3eef2133d44d0190ec08ac6d6eba4913d031294e9R734) and changed to v2